### ISSUE TITLE
~mta cleanup

### ISSUE THREAD

**Issue by TheOriginalSoni** [kevslinger/bot-be-named#17]
add archive 1, archive 2, archive 3, archive 4 as acceptable archive suffixes for mta (automated?)

**Comment by petecao** [comment]
Is this like choosing a archive category? If so, done with mtalion

**Comment by TheOriginalSoni** [comment]
Yeah I think it's good enough for now, but we might have to re-open this later at some point to figure out a clever-er solution. I'm not fully happy with mtalion solution
